tcf_website/templates/department/department.html

```diff
@@ -17,16 +17,43 @@
 
     <div class="subdepartments mt-3">
         {% for sd in subdepartments %}
-        <div class="subdepartment container-lg {% if not sd.has_current_course %}old{% endif %}">
+        <div class="subdepartment container-lg">
             <h1 class="my-3">{{ sd.name }}</h1>
             <ul class="course-list" id="courses-sd-{{sd.id}}">
             </ul>
             <div class="text-center" id="spinner-sd-{{sd.id}}">
-                <div class="spinner-border" role="status" >
+                <div class="spinner-border" role="status">
                     <span class="sr-only">Loading...</span>
                 </div>
             </div>
         </div>
+        <!-- Card for if there are no courses in this subdepartment in general-->
+        {% if not sd.recent_courses %}
+        <div class="new card col p-5 text-center">
+            <div class="card-body">
+                <h4 class="card-title">
+                    Looks like this subdepartment has no courses that are taught as of now. <i
```

Comment:
I'd change this to "hasn't been offering courses for a while" or something like that because IMO "as of now" is a little vague.